pysipp/launch.py

```diff
@@ -178,3 +180,14 @@ async def finalize(runner, agents, timeout):
         raise SIPpFailure(msg)
 
     return cmds2procs
+
+
+def finalizer(finalize_coro, runner, agents):
+    def with_timeout(timeout):
+        return trio.run(
+            partial(finalize_coro, timeout=timeout),
```

Comment:
This partial also feels weird.
Maybe put the positional args inside it as well?

Reply:
it's because trio.run passes only args and not keyword args but sure